libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/OAuthInput.cs

```diff
@@ -330,7 +316,7 @@ private async Task SendOAuthCardAsync(DialogContext dc, IMessageActivity prompt,
 
         private Task<PromptRecognizerResult<TokenResponse>> RecognizeTokenAsync(DialogContext dc, CancellationToken cancellationToken)
         {
-            var settings = new OAuthPromptSettings { ConnectionName = ConnectionName.GetValue(dc.State) };
+            var settings = new OAuthPromptSettings { ConnectionName = ConnectionName?.GetValue(dc.State) };
```

Comment:
This is not directly this PR, however, this looks like a bug so we'll make it consistent with what we have in this PR.